.circleci/config.yml

```diff
@@ -195,11 +196,15 @@ jobs:
             then
               ./gradlew --no-daemon expandRefTests
             fi
+      - run:
```

Comment:
Why are we separating the compiling of reference tests from the Assemble command?

Reply:
Because we can't compile the reference tests until we've generated the sources and can't do that until we've actually got the eth2.0-reference-test stuff expanded.  And we only do that in the referenceTest job.